service_configuration_lib/spark_config.py

```diff
@@ -1130,27 +1034,22 @@ def get_spark_conf(
     :param aws_creds: the aws creds to be used for this spark job. If a key triplet is passed,
         we configure a different credentials provider to support this workflow.
     :param extra_volumes: extra files to mount on the spark executors
-    :param extra_docker_params: extra docker parameters to launch the spark executor
```

Comment:
This is interesting. 
Ok for now. But something to keep in the radar.
This looks like a useful param to have even in spark on k8s. I wonder why was it supported for cluster-manager k8s.